src/adapter/graphql/src/utils.rs

```diff
@@ -124,13 +124,44 @@ pub(crate) fn check_logged_account_id_match(
         }
     };
     Err(GqlError::Gql(
-        async_graphql::Error::new("Account datasets access error")
+        async_graphql::Error::new("Account access error")
             .extend_with(|_, eev| eev.set("account_id", account_id.to_string())),
     ))
 }
 
 ///////////////////////////////////////////////////////////////////////////////
 
+pub(crate) async fn check_access_token_access(
```

Comment:
Maybe "check_access_token_valid". Acces access is a bit hard :-)

Reply:
Yeah, agree. Changed